www/src/components/PropsTable.jsx

```diff
@@ -8,18 +8,18 @@ const renderType = ({ name, value }) => { // eslint-disable-line
     case 'shape':
       return <ShapeProp shape={value} />;
     case 'union':
-      return <span>{value.map(v => renderType(v))}</span>;
+      return <span key={name}>{value.map(v => renderType(v))}</span>;
```

Comment:
These `key={name}` additions get rid of the missing keys warning in PropsTable renderer.